### ISSUE TITLE
How to load css/scss based on different pages.

### ISSUE THREAD

**Issue by hirenramoliyaiwb** [nuxt/framework#7750]
### Describe the feature

I want to load some css/scss files only on some pages not globally so, is there any way to import/load conditionally 

### Additional information

- [ ] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by sevillaarvin** [comment]
Hi, does `useHead` meet your requirements?

https://v3.nuxtjs.org/api/composables/use-head#add-external-css

**Comment by hirenramoliyaiwb** [comment]
@sevillaarvin This is right but in that tailwind classes not working like the @apply keyword, and for that, i have to add that particular file into nuxt config file and that leads to a global load in all pages.

**Comment by manniL** [comment]
I think this should happen automatically when using (S)CSS in the component/page only and when the content is big enough for an own file (worth doing an extra HTTP request)
